### DISCUSSION TITLE
Atom does not appear in the Win 10 program listing

### DISCUSSION THREAD

**Discussion by cybervigilante** [atom/atom#22718] [Q&A]
Is there any way to install it into the program listing? It's installed since I see the option to uninstall it in programs. Luckily, I pinned it to the launch bar before I closed it or it would be impossible to find at all. But I'm trying to not clutter the launch bar.

**Comment by icecream17** [comment]
https://github.com/atom/atom/issues/14020

Maybe you can try copying/pasting `atom.exe` from `C:\Users\[profile]\AppData\Local` (AppData is a hidden folder) to `C:\Program Files`?
(This is just a guess)

**Reply by cybervigilante** [reply]
Alas, all that's in there is update.exe, and it also isn't in program files. It's like they're hiding it on purpose 😃 Turns out it's in C:\Users\cyber\AppData\Local\JuliaPro-1.5.4-1\app-1.55.0\atom.exe so I'll go fiddle with that. So I found the start menu folder, found Atom, made a shortcut for it, pasted that into the start menu folder - and atom still doesn't appear. I guess the folks at atom hate windows. Anyway, I'm just ditching it. I have iJulia notebook and I've found that if I want a longer program than repl fun, all I have to do is run notepad from the REPL to write a jl file then include it. For more complex stuff I'll use iJulia. And if I want to do something big, install Visual Studio. 

Ah, the mystery solved. Atom is listed under Julia Pro. Click on that and it starts Atom.
